### ISSUE TITLE
Add possibility to create a pie chart, bar chart with horizontal bars

### ISSUE THREAD

**Issue by KirylBubovich** [JetBrains/lets-plot#361]
It would be nice to have pie chart and bar chart with horizontal bars support

**Comment by alshan** [comment]
Yes, we are planning to have horizontal bars (and some other directional geometries).

As for pie char, this more difficult, but we are planning to have "waffle plot" which can work as a replacement for pie chart.

**Comment by KirylBubovich** [comment]
I would like to ask a couple of questions
1) It would be interesting to know where you see the difficulty of adding a pie chart. I have some sort of workaround. I understand that if I save this to svg then the image size will be quite large. Can you please describe other disadvantages of my approach or maybe you can suggest your own 
![image](https://user-images.githubusercontent.com/47569841/115769578-5d519300-a3b4-11eb-9eb1-669fa981dafe.png)

**Comment by KirylBubovich** [comment]
2) The second question is a bit off-topic, but still I'm interested in how to do it. Suppose I want to make a chart like this. I managed to do something similar using facet. Is it possible to achieve this effect without using facets? Is it possible to somehow control the distance between bars?
![image](https://user-images.githubusercontent.com/47569841/115770123-f54f7c80-a3b4-11eb-9ccf-892f324719e0.png)
![image](https://user-images.githubusercontent.com/47569841/115770163-00a2a800-a3b5-11eb-8877-2dd4d328939d.png)
Thanks in advance for your answers!

**Comment by alshan** [comment]
On the 2nd, try:
```
geomBar(stat..., position = Pos.dodge) { x = "category"; ...
```

**Comment by alshan** [comment]
In Lets-Plot everything boils down to points in some coordinate system.
To implement idiomatic Pie Chart we would have to use polar coordinate system which is not yet supported.
Adding support for polar coordinate system is difficult (but also very cool).

But you are right: noneidiomatic solution for pie chart can be done with lesser efforts. Your solution is original but a bit extreme. Could you try to build sectors using `geomPolygon`?
